frontend/src/metabase/visualizations/types/visualization.ts

```diff
@@ -156,18 +157,34 @@ export type ColumnSettingDefinition<TValue, TProps = unknown> = {
     col: DatasetColumn,
     settings: OptionsType,
     onChange: (value: TValue) => void,
-    extra: { series: Series },
+    extra: {
```

Comment:
not sure if `extra` is the same as the one below?

Reply:
Seems it also includes settings according to `frontend/src/metabase/visualizations/lib/settings/nested.js`